src/dags/common/url.py

```diff
@@ -0,0 +1,74 @@
+#  Copyright 2019 SURF.
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#          http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+from __future__ import annotations
+
+from typing import Mapping
+from urllib.parse import urlencode
+
+
+class URL(str):
```

Comment:
Nice class. Maybe we could put it in schema-tools lib?

Reply:
De toepasbaarheid is breder dan schema-tools. Heb daarom even overwogen om er een apart PyPi package van te maken. Maar dat voelt zo Javascript-esque voor zo weinig code.

Reply:
Ik laat hem nog even hier staan en dan kunnen we het bespreken (slack, standup, standdown)

Reply:
+1 voor een aparte package. Erg coole lib. Beetje net als `cached_property` een aparte package is geweest.
Je kan het nog generieker trekken, als een `string_types`  package waarin je meer incompatibile types hebt, zoals `SQL` and `ShellCommand`

Reply:
Ik maak er morgen even een aparte package van.

Reply:
Done: https://pypi.org/project/more-ds/

Heb:
- De `URL` class uit dataservice-airflow gehaald. 
- Een dependency opgenomen naar `more-ds`
- Import aangepast.

Tav naamgeving @vdboor Ik wil nog een andere data structuur aan `more-ds` toevoegen (ook van SURF tav `VLAN ranges`), en dat heeft niet echt iets met strings te maken. Dus vandaar dat ik een wat andere naam heb gekozen.

Oh, en `more-ds` is een repo onder m'n eigen naam, want in m'n eigen tijd gedaan.